_episodes/01-why-bother-performance.md

```diff
@@ -20,16 +20,10 @@ keypoints:
 
 ## What is software performance?
 
-Before getting into the software side of things, lets take a few steps back. Performance is a generic term that applies to may factors of our lives, from physical to mental and engineering. However with software performance the emphasis isn't always on the most powerful machine, it is how you best utilise the power that you have. Say you are the chef in a restaurant and every dish that you do is perfect. You would be able to produce a set 7 course meal for a table of 3-6 without too much difficulty. If your restaurant is full though, it becomes more difficult, as people would be waiting many hours for their food. However if you had a team of 6 additional chefs with you, assisting as well as communicating with each other, delegating tasks, you have a much higher chance of getting the food out on time and coping with a large workload, likely with each concentrating on doing one course 
+Before getting into the software side of things, lets take a few steps back. Performance is a generic term that applies to may factors of our lives, from physical to mental and engineering. However with software performance the emphasis isn't always on the most powerful machine, it is how you best utilise the power that you have. Say you are the chef in a restaurant and every dish that you do is perfect. You would be able to produce a set 7 course meal for a table of 3-6 without too much difficulty. If your restaurant is full though, it becomes more difficult, as people would be waiting many hours for their food. However if you had a team of 6 additional chefs with you, assisting as well as communicating with each other, delegating tasks, you have a much higher chance of getting the food out on time and coping with a large workload, likely with each concentrating on doing one course.
 
 It's the same with software performance, if you have one process running doing one thing there is usually never an issue. But if you have one doing several things, it will obviously take much longer. Furthermore if you hade several workers working on several tasks, it will take a fraction of the time. That is the essence of software performance, using the resources you have to their best capabilities.
```

Comment:
```
It's the same with software performance, if you have one process running doing one thing there is usually never an
issue. But if you have one doing several things, it will obviously take much longer. If you have access to several workers to whom you can assign tasks or effort, it will take a fraction of the time. That is the essence of software performance, using the resources you have to their best capabilities.
```